src/newtondynamics_physics/factors.h

```diff
@@ -24,10 +24,15 @@
  */
 #pragma once
 
-#include "../Universe.h"
-#include "activetypes.h"
+#include <array>
+#include <cstdint>
 
-#include <unordered_map>
+namespace ospnewton
+{
 
-using MapSatToEnt_t = std::unordered_map<osp::universe::Satellite,
-                                         osp::active::ActiveEnt>;
+// Each rigid body is given 64 bits to enable/disable forces
+// These determine which physics calculations are required for a certain
+// rigid body, such as gravity, thurst, or aerodynamics.
+using ForceFactors_t = std::array<uint64_t, 1u>;
```

Comment:
Bit lost.

This is a 64bit wide bitfield?

I recommend using either a proper enum holding the designated bits, or use a struct that holds a uint64_t that's broken out into bit fields at the language level. e.g.

struct Forces
{
    uint64_t m_gravity : 1;
    uint64_t m_solarWind : 1;
    uint64_t m_wind : 1;
    uint64_t m_explosions : 1;
    uint64_t m_unspecified : 60;
}

Reply:
The forces are assigned at runtime, I guess I should include this in the comment